File: fjt.api/models/GenericFiles.js
const Sequelize = require('sequelize');

module.exports = (sequelize) => {
    const GenericFiles = sequelize.define('GenericFiles', {
        gencFileID: {
            allowNull: false,
            autoIncrement: true,
            primaryKey: true,
            type: Sequelize.INTEGER
        },
        gencFileName: {
            type: Sequelize.STRING,
            allowNull: true,
            validate: {
                notEmpty: true,
                len: [1, 100]
            }
        },
        gencOriginalName: {
            type: Sequelize.STRING,
            allowNull: true,
            validate: {
                notEmpty: true,
                len: [1, 255]
            }
        },
        gencFileDescription: {
            type: Sequelize.TEXT,
            allowNull: true
        },
        gencFileExtension: {
            type: Sequelize.STRING,
            allowNull: true,
            validate: {
                notEmpty: true,
                len: [1, 100]
            }
        },
        gencFileType: {
            type: Sequelize.STRING,
            allowNull: true,
            validate: {
                notEmpty: true,
                len: [1, 100]
            }
        },
        tags: {
            type: Sequelize.STRING,
            allowNull: true
        },
        isDefault: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        refTransID: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        entityID: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        gencFileOwnerType: {
            type: Sequelize.STRING,
            allowNull: true,
            validate: {
                notEmpty: true,
                len: [1, 255]
            }
        },
        isDeleted: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        isActive: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        genFilePath: {
            type: Sequelize.STRING,
            allowNull: true
        },
        createdBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        updatedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        deletedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        isShared: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        fileGroupBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        refParentId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        fileSize: {
            type: Sequelize.BIGINT,
            allowNull: false
        },
        isDisable: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        disableOn: {
            type: Sequelize.DATE,
            allowNull: true
        },
        disableBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        disableReason: {
            type: Sequelize.STRING,
            allowNull: true
        },
        refCopyTransID: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        refCopyGencFileOwnerType: {
            type: Sequelize.STRING,
            allowNull: true
        },
        createByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        updateByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        deleteByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        isRecycle: {
            type: Sequelize.BOOLEAN,
            allowNull: true,
            defaultValue: false
        }
    }, {
            paranoid: true,
            tableName: 'genericfiles'
        });

    GenericFiles.associate = (models) => {
        GenericFiles.belongsTo(models.Equipment, {
            as: 'equipmentfiles',
            foreignKey: 'refTransID'
        });
        GenericFiles.belongsTo(models.GenericFolder, {
            as: 'generic_folder',
            foreignKey: 'refParentId'
        });
    };

    return GenericFiles;
};

File: fjt.pricingservice/MySqlDBModel/genericfiles.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace fjt.pricingservice.MySqlDBModel
{
    [Table("genericfiles")]
  public class genericfiles
    {
        [Key]
        public int gencFileID { get; set; }
        public string gencFileName { get; set; }
        public string gencOriginalName { get; set; }
        public string gencFileDescription { get; set; }
        public string gencFileExtension { get; set; }
        public string gencFileType { get; set; }
        public string tags { get; set; }
        public bool isDefault { get; set; }
        public int refTransID { get; set; }
        public string gencFileOwnerType { get; set; }
        public string gencFilePath { get; set; }
        [Required]
        public bool isDeleted { get; set; }
        public bool isActive { get; set; }
        [StringLength(255)]
        public string createdBy { get; set; }
        public DateTime createdAt { get; set; }
        [StringLength(255)]
        public string updatedBy { get; set; }
        public DateTime? updatedAt { get; set; }
        [StringLength(255)]
        public string deletedBy { get; set; }
        public DateTime? deletedAt { get; set; }
        public bool isShared { get; set; }
        public string fileGroupBy { get; set; }
        public int refParentId { get; set; }
        public long fileSize { get; set; }
        public bool isDisable { get; set; }
        [StringLength(255)]
        public string disableBy{ get; set; }
        public DateTime? disableOn { get; set; }
        public string disableReason { get; set; }
        public int refCopyTransID { get; set; }
         public string refCopyGencFileOwnerType { get; set; }
        public int createByRoleId { get; set; }
        public int updateByRoleId { get; set; }
        public int deleteByRoleId { get; set; }








    }
}
